ebpf/testutil/testdata.go

```diff
@@ -0,0 +1,16 @@
+package testutil
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func InitGitSubmodule(t *testing.T) {
```

Comment:
🤣

Reply:
everybody hates to use git submodules so I try to pretend it is not there :D